src/lib.rs

```diff
@@ -92,27 +159,33 @@ pub struct EditorConfigINIAST {
 
 #[derive(Debug)]
 struct Section {
-	name: String,
+	header: SectionHeader,
 	body: Vec<Item>,
 }
 
 #[derive(Debug)]
-struct Property {
+struct SectionHeader {
 	name: String,
-	value: String,
-	newline: String,
+	raws: Raws,
 }
 
-// #[derive(Debug)]
-// struct BlankLine {
-// 	newline: String,
-// 	// raws.before
-// }
+#[derive(Debug)]
+struct Raws {
+	before: Option<String>,
```

Comment:
We could also talk about removing the `Option` from all of these and just always making sure it's at least an empty string.

Reply:
Probably, using an Option points out there is the existence of a value expected or not. Having an empty String is probably misleading - sure in this simple case probably not.